src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
-<html lang="en">
-  <head>
+<html lang="en" class="page">
+
+  <head class="head">
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+
+  <body class="page__body">
+    <div class="group page__group">
+      <section class="stars stars--0">
+        <div class="stars__star stars__star--icon"></div>
+        <div class="stars__star stars__star--icon"></div>
+        <div class="stars__star stars__star--icon"></div>
+        <div class="stars__star stars__star--icon"></div>
+        <div class="stars__star stars__star--icon"></div>
```

Comment:
```suggestion
      <section class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```
Stars elements shouldn't have any modifiers
![image](https://github.com/mate-academy/layout_stars/assets/96409563/551d0db9-6965-43d7-bc46-ae4b12efd1ea)